lib/cyclonedx/report_node_modules_cyclonedx.rb

```diff
@@ -1,7 +1,18 @@
 module Cyclonedx
   class ReportNodeModules < Base
+    TYPE = "npm".freeze
```

Comment:
@saron-deb If you feel that Python, Rust, and Go can also fit this style it could be a win to cut down some code.

Reply:
Yep they definitely can, will make that change (in a separate PR?)

Reply:
Yep separate PR once this one is merged 👍